ligo/src/registryDAO.mligo

```diff
@@ -166,7 +166,9 @@ let registry_DAO_decision_lambda (proposal, extras : proposal * contract_extra)
     : operation list * contract_extra =
   let propose_param : propose_params = {
     frozen_token = proposal.proposer_frozen_token;
-    proposal_metadata = proposal.metadata
+    proposal_metadata = proposal.metadata;
+    name = proposal.name;
+    description = proposal.description;
     } in
   let proposal_key = to_proposal_key (propose_param, proposal.proposer) in
```

Comment:
Similarly this seems incorrect to me.

The discussion linked in the description indeed is about this point and the fact that `toProposalKey` uses the `propose_param` which has now the problem of including `name` and `description` (note: the issue remains if these are in the `proposal_metadata`).

I figured that this was bad because these are metadata and shouldn't really influence the proposal size calculation (which they do ATM), however it was argued that it wouldn't be that bad and that the parameters can be tweaked to account for those. What do you think?

Reply:
If for whatever reason the users think it should be accounted for, it's already there. If not, then they can tweak the parameters or leave these fields empty, so I think it makes sense to account for them. But I don't really have much of a strong opinion towards one option or the other, so perhaps the simpler one is best, I think.

Reply:
Indeed, the parameters should be flexible enough to be tweaked or not for those, which leaves the choice to the user as well as make the default calculation much simpler.

So agreed, no modification to the `propose_params.frozen_token` check will need to be done here.